guides/promotions/promotion-actions.md

```diff
@@ -0,0 +1,71 @@
+# Promotion actions
+
+The `Spree::PromotionAction` model defines an action that should occur if the
+promotion is activated and eligible to be applied. There can be multiple
+promotion actions on a promotion.
+
+Typically, a promotion action could be free shipping or a fixed percentage
+discount.
+
+A promotion action calculates the discount amount and creates a
+`Spree::Adjustment` for the promotion. The adjustment then adjusts the price of
+an order, line item, or shipment.
+
+With the exception of `Spree::Promotion::Actions::FreeShipping`, promotion
+actions have a configurable base calculator. This gives you and store
+administrators flexibility for choosing how a promotion amount is calculated.
+
+<!-- TODO:
+  Once calculator documentation exists, link to it in the above paragraph so
+  there's more context for anyone wondering what a "base calculator" is in
+  Solidus.
+  
+  Similarly, we should link to the adjustments documentation once it's merged.
+-->
+
+## Available promotion action types
+
+The following classes are [subclasses of the `Spree::Promotion::Actions`
+model][promotion-actions]:
+
+- `CreateAdjustment`: Creates a single adjustment for the current
+  `Spree::Order`.
+- `CreateItemAdjustments`: Creates an adjustment for each `Spree::LineItem` in
+  the current order.
+- `CreateQuantityAdjustments`: Creates per-quantity adjustments. For example,
+  you could create an action that gives customers a discount on the first three
+  t-shirts that they order.
+- `FreeShipping`: Creates an adjustment that subtracts all shipping charges.
```

Comment:
I would use "negates" instead of "subtracts", but that might just be personal preference.